src/main/java/com/dungeoncrawler/Javiarenka/equipment/Equipment.java

```diff
@@ -0,0 +1,37 @@
+package com.dungeoncrawler.Javiarenka.equipment;
+
+public class Equipment {
+    private String name;
+    private double weight;
+    private double price;
```

Comment:
Zastanawiam się czy pola price nie zmienic na int, ale to pewnie zależy czy będziemy chcieli żeby ceny wyrażały się w tysiącach monet, czy raczej w formacie xx.yy. Zastanawiałem się jeszcze nad tym, żeby rozszerzyć to pole na trzy pola priceGold, priceSilver, price Copper, ale to pewnie w przyszłości.